tests/integrations/unionTooLargeToRepresent/__app.ts

```diff
@@ -322,14 +322,14 @@ export const types = [
   unionType({
     name: 'MediumUnionType',
     definition(t) {
-      t.members(...typeNames.slice(50, 150))
+      t.members(...(typeNames as any).slice(50, 150))
       t.resolveType(() => null)
     },
   }),
   unionType({
     name: 'MediumUnionTypes',
     definition(t) {
-      t.members(...typeNames.slice(0, 100))
+      t.members(...(typeNames as any).slice(0, 100))
```

Comment:
Is this to make `ts-jest` happy or IDE? In my case VSCode was correctly interpreting the typings here. And given your mentioned solution to make ts-jest ignore these integration tests then I don't think we need these `any` changes here anymore?

Also these `any` would affect the actual integration test too, which isn't ideal.

Reply:
I updated the TS integration tester to use a local tsconfig.json if it is present. This way we can satisfy the IDE and the integration twice at once.